roles/StackStorm.st2/tasks/main.yml

```diff
@@ -100,3 +100,15 @@
     state: started
   loop: "{{ st2_services }}"
   tags: st2
+
+# https://github.com/ansible/ansible/issues/41313#issuecomment-520891625
+- name: Flush handlers to prepare to run st2 there are packs to install
+  include_tasks: flush_handlers.yml
+  when: st2_packs is defined
+
+- name: Install ST2 add-on Packs
+  import_tasks: packs.yml
+  when: st2_packs is defined and
+        ((ansible_distribution == "RedHat" and ansible_distribution_major_version == '7') or
+        (ansible_distribution == "Ubuntu"))
+  tags: st2, st2_packs, skip_centos_6
```

Comment:
Additionally, if target OS is `el6` can we print an error message indicating that `st2_packs` is not supported with EL6?

It could be done with `fail` Ansible task + `ignore_errors: yes`.

Reply:
done